Self-Assessment test had been added too bottom navbar

Kurubaran-18 opened the pull request:
Added dependency ( font_awesome_flutter - For stethoscope Icon) & (delayed_display - For Chat View in Self Assessment Test)

Added a dart file in (lib/ui/screens/health.dart)
Added a directory (health) in lib...& added few files in that directory for self-assessment test

Removed Facts from bottomNavigation and added it to the appbar of Dashboard.

Commit:
Added dependency ( font_awesome_flutter - For stethoscope Icon) & (delayed_display - For Chat View in Self Assessment Test)
Added a dart file in (lib/ui/screens/health.dart)
Added a directory (health) in lib...& added few files in that directory for self-assessment test

Removed Facts from bottomNavigation and added it to the appbar of Dashboard.

ghost commented:
# Added Screenshots

![Screenshot_20200711-175640](https://user-images.githubusercontent.com/63589833/87224214-12283f80-c3a1-11ea-88bd-f278172e51b9.jpg)
![Screenshot_20200711-175656](https://user-images.githubusercontent.com/63589833/87224215-181e2080-c3a1-11ea-9c2f-7ad5db355fba.jpg)
![Screenshot_20200711-180002](https://user-images.githubusercontent.com/63589833/87224217-1d7b6b00-c3a1-11ea-98ad-17dcfb8306b7.jpg)
![Screenshot_20200711-180008](https://user-images.githubusercontent.com/63589833/87224218-21a78880-c3a1-11ea-9608-f173cc841896.jpg)

Commit:
Issue Fix

On Facts Added a Back Button to go back to HomeScreen.....
Issue: OnClicking back button, the app gets closes due to pop.
Fix: Added a Route to HomeScreen
